src/libs/actions/PersonalDetails.js

```diff
@@ -107,9 +107,10 @@ function formatPersonalDetails(personalDetailsList) {
 
 /**
  * Get the personal details for our organization
+ * @returns {Promise}
  */
-function fetch() {
-    API.Get({
+function fetchPersonalDetails() {
```

Comment:
NAB: Why do you rename fetch? Since it's under PersonalDetails, it implies it's personal details what it's retrieving.

Reply:
I guess this is a personal preference to put in my personal details ? 😖  

When we have a generic export like `fetch()` it encourages renaming imports and ultimately makes it hard to track down usages of things e.g. take several examples:

**Here we can rename this to whatever and it makes finding things hell**
```js
import {fetch as fetchDetails} from './PersonalDetails';
```
```js
import {fetch as fetchThePersonalDetails} from './PersonalDetails';
```
```js
import * as PersonalDetails;
PersonalDetails.fetch();
```

**Here there is no confusion and if we search for the name we'll always find and it's less likely someone will change the name to avoid a conflict with another generic `fetch()`.**

```js
import * as PersonalDetails from './PersonalDetails';
PersonalDetails.fetchPersonalDetails();
```
```js
import {fetchPersonalDetails} from './PersonalDetails';
```

Searching for `fetchPersonalDetails` in the latest case would bring up all usages.

Reply:
Gotcha, it now makes sense!